src/definitions/modules/dropdown.less

```diff
@@ -565,13 +565,23 @@ select.ui.dropdown {
     padding: @selectionItemPadding;
     white-space: normal;
     word-wrap: normal;
+    z-index: 0;
   }
 
   /* User Item */
   .ui.selection.dropdown .menu > .hidden.addition.item {
     display: none;
   }
 
+  /* User addition item */
```

Comment:
To "support" IE11/old Edge, we should wrap this by `@supports(position: sticky)`